app/controllers/messages_controller.rb

```diff
@@ -0,0 +1,33 @@
+class MessagesController < ApplicationController
+
+  def create
+    @message = Message.new(message_params)
+    @match = Match.find_by(id: params[:match_id])
+    @message.user = current_user
+    @message.match = @match
```

Comment:
I'd redo this so that message_params does
```
def message_params
  params.require(:message).permit(:content).merge(user: current_user)
end
```

then in this snippet do
```
@match = Match.find_by(id: params[:match_id])
message = @match.message.new(message_params)
```